README.md

```diff
@@ -120,6 +120,22 @@ Then, to create an administrator for the organization:
 
 which returns the `user_id`.
 
+### Resetting the Database When Upgrading Arlo
+
+If you're upgrading Arlo, right now the only way is to destroy and
```

Comment:
What exactly does upgrading mean? Does it mean moving between versions where the database schema changed? Are these instructions standing in place of a strategy for db migrations?

Reply:
exactly, we don't have db migrations yet and we've explicitly chosen not to have them until we really need them. Which might be soon :)